sprompt.cpp

```diff
@@ -57,7 +57,9 @@
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
-#include <unistd.h>
+#ifndef _WIN32
+#    include <unistd.h>
+#endif
 
```

Comment:
Should work without it (cmake should generate it). Many other places with unistd.h.